pipelines/resources/deploy-ztp-vsphere.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
this file should be placed on folder contrib like:
[https://github.com/rh-ecosystem-edge/ztp-pipeline-relocatable/tree/33f65288208908623e96fd82269bd381c0ba6560/pipelines/resources/contrib](https://github.com/rh-ecosystem-edge/ztp-pipeline-relocatable/tree/33f65288208908623e96fd82269bd381c0ba6560/pipelines/resources/contrib)
this is the way to use contrib resources in our pipeline

Reply:
also there, you've got a template file created just to copy and paste modifying the resources and deployments you need to use in your case